chrome-extension/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Lobster Ice Cream",
+  "version": "0.1",
+  "description": "Trip idea clipper.",
+  "manifest_version": 2,
+  "browser_action": {
+    "default_title": "Lobster Ice Cream Sidebar",
+    "default_icon": "./assets/lobster.jpg",
```

Comment:
Is this being used anywhere? Consider removing it for now if not.

Reply:
That would also include deleting lobster.jpg

Reply:
It's supposed to be the icon that shows up for the chrome extension on the tab (https://developer.chrome.com/extensions/browserAction) but it's not showing rn. Might be something wrong with the image

Reply:
I will remove for now and leave a todo?

Reply:
Ahhhh okay. It might be that it's too big? I agree with removing for now and leaving a todo since it doesn't work

Reply:
yeah i think it might be that- to big. but ok, will do!